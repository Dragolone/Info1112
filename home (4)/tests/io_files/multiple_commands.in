pwd
echo "Testing"
ls -m
exit
