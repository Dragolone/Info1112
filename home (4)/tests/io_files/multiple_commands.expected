/home/tests
Testing
io_files, run_tests.sh
